lammps/doc/src/fix_colvars.txt

```diff
@@ -98,8 +98,15 @@ fix to add the energy change from the biasing force added by the fix
 to the system's potential energy as part of "thermodynamic
 output"_thermo_style.html.
 
-The {fix_modify add_config <config file>} option allows to add settings
-from additional config files to the colvars module.
+The {fix_modify configfile <config file>} option allows to add settings
+from an additional config file to the colvars module. This option can
+only be used, after the system has been initialized with a "run"_run.html
```

Comment:
@akohlmey I had overlooked this particular note: `fix_modify` does not have this restriction for other fixes, and the changes generated by a `configfile` sub-command are the same that would be done by the initial setup.